### PR TITLE
feat: support fast build tiflash at nightly

### PR DESCRIPTION
**Pull request by lijie0123**
Why:
- enable experimental tiflash pipeline at nightly build
- fix tiflash pipeline

How:
- can choose skip tiflash in the optimization pipeline, avoid conflict

close: https://github.com/PingCAP-QE/ci/issues/2247

### CODE REVIEW

File: jenkins/jobs/cd/build_tiflash/master.groovy [hunk 1]
```diff
@@ -21,7 +21,7 @@ pipelineJob('build-tiflash-master') {
         }
     }
     parameters {
-        stringParam('GitHash', '7bd02a86e08f5077c69299ac048a700fcb68507f', 'the git tag or commit')
+        stringParam('GitHash', '', 'the git tag or commit')
         stringParam('Version', 'v7.3.0', 'important, the Version for cli --Version and profile choosing, eg. v6.5.0')
```
Comment (purelind) [inline jenkins/jobs/cd/build_tiflash/master.groovy:25]: Suggested `Version` default value is kept as an empty string.

Comment (lijie0123) [reply jenkins/jobs/cd/build_tiflash/master.groovy]: fixed

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**